src/init.rs

```diff
@@ -53,6 +55,7 @@ impl Init {
         self.create_project(&args.name)?;
 
         self.init_config(&chip, &probe_target_name)?;
+        self.init_debug_config(&chip, &probe_target_name, &args.name)?;
```

Comment:
I'm not a fan of generating this file unconditionally. Many people (myself included) *do not* use VSCode and won't want our workspace cluttered with a `.vscode` folder.

An additional flag like `--vscode-launch` would be good, I think.